scripts/spark2diffs/RegexParser.diff

```diff
@@ -2,7 +2,7 @@
 <  * Copyright (c) 2022, NVIDIA CORPORATION.
 ---
 >  * Copyright (c) 2021-2022, NVIDIA CORPORATION.
```

Comment:
this shouldn't have changed

Reply:
Yeah, it's strange that it shows a difference here.
I compared locally and it only shows the line5.

Reply:
I think it just shows the content up and down around the changed place.
The content contains the similar format of a diff file, then github shows it as a difference.
But there's no +/- mark in the lines.